bcda/web/api.go

```diff
@@ -204,6 +216,91 @@ func bulkRequest(t string, w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusAccepted)
 }
 
+func check429(jobs []models.Job, types []string, w http.ResponseWriter) ([]string, bool) {
```

Comment:
This seems overly complex.  What if we give a `429` regardless of the type of job they have requested?
They have the freedom to request `/Patient` (ALL) and get everything with one request.

Reply:
workin on that confluence page..stay tuned :)